src/lib/containers/entity/annotations/AnnotationEditorUtils.ts

```diff
@@ -0,0 +1,97 @@
+import { AjvError } from '@sage-bionetworks/rjsf-core'
```

Comment:
The functions in this file were pulled out of the SchemaDrivenAnnotationEditor component file. They are also now tested separately.